src/Microsoft.AspNet.FileProviders.Physical/project.json

```diff
@@ -0,0 +1,20 @@
+{
+    "version": "1.0.0-*",
+    "description": "Implementation of ASP.NET 5 physical file provider abstractions.",
+    "dependencies": {
```

Comment:
Don't you need a dependency on Interfaces here? Common gets you one transitively, but it's only a build-time dependency.  Same for Embedded.